seqr/views/apis/report_api_tests.py

```diff
@@ -375,15 +375,15 @@ def test_anvil_export(self, mock_google_authenticated,  mock_zip, mock_analyst_g
             'entity:discovery_id', '01-subject_id', '02-sample_id', '03-Gene', '04-Gene_Class',
             '05-inheritance_description', '06-Zygosity', '07-variant_genome_build', '08-Chrom', '09-Pos',
             '10-Ref', '11-Alt', '12-hgvsc', '13-hgvsp', '14-Transcript', '15-sv_name', '16-sv_type',
-            '17-significance']))
+            '17-significance', '18-discovery_notes']))
```

Comment:
We need to add a test case for this, sadly. I know that involves adding more fixture data which is frustrating to test but it is important. I think its okay to add either a MNV OR a complex SV, I don't think we need both

Reply:
Okay, I will add some tests.

Reply:
A test has been added.